composer/algorithms/gradient_clipping/gradient_clipping.py

```diff
@@ -35,7 +36,31 @@ def apply_gradient_clipping(parameters: Union[torch.Tensor, Iterable[torch.Tenso
             to (for 'value'), what values to clip the gradient norms to (for 'norm'), and
             threshold by which if grad_norm / weight_norm is greater than this threshold then
             scale gradients by this threshold * (weight_norm / grad_norm) (for 'adaptive').
+        fsdp_enabled (bool): Bool of if the model is a FSDP model or not.
     """
+    if fsdp_enabled:
+        from torch.distributed.fsdp import FullyShardedDataParallel
+        for module in model.modules():
+            if isinstance(module, FullyShardedDataParallel):
+                # We can only call grad clip on the parent instance, so we iterate through all
```

Comment:
does this require torch 1.13 as we do everywhere else with fsdp?

Reply:
I think no, but in case I'll add a check for the PyTorch version